.bash_aliases

```diff
@@ -0,0 +1,4 @@
+# for Proficio
+alias leftConfig='cp ~/libbarrett/proficio_sandbox/configurations/wam3.conf.left /etc/barrett/wam3.conf; cp ~/libbarrett/proficio_sandbox/configurations/zerocal.conf.left /etc/barrett/calibration_data/wam3/zerocal.conf; cp ~/libbarrett/proficio_sandbox/configurations/calibration.conf.left /etc/barrett/calibration.conf; cp ~/libbarrett/proficio_sandbox/configurations/gravitycal.conf.left /etc/barrett/calibration_data/wam3/gravitycal.conf'
```

Comment:
These .left and .right files should be in /etc/barrett, not ~/libbarrett. We shouldn't depend on the user checking out the code into ~/libbarrett.